app/components/Wallet/Wallet.js

```diff
@@ -86,7 +94,8 @@ const Wallet = ({
                 </span>
               </h1>
               <span className={styles.usdValue}>
-                ≈ ${usdAmount ? usdAmount.toLocaleString() : ''}
+                ≈ {currentTicker[ticker.fiatTicker].symbol}
+                {usdAmount ? usdAmount.toLocaleString() : ''}
```

Comment:
`usdAmount` -> `fiatAmount`

Reply:
In `btcToUsd` we use `numberWithCommas(amount)` to format fiat amount. We should probably use this everywhere. Maybe we need a general `formatFiatAmount` method that we call when we want to format fiat amounts rather than calling `toLocaleString` 

Alternatively, it looks like the `toLocaleString` method actually has built in support for currencies.

See

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#Parameters